deps/config/types.go

```diff
@@ -365,6 +367,9 @@ type CurioSubsystemsConfig struct {
 	// The maximum amount of indexing and IPNI tasks that can run simultaneously. Note that the maximum number of tasks will
 	// also be bounded by resources available on the machine. (Default: 8 - unlimited)
 	IndexingMaxTasks int
+
+	// EnableMarketBalanceManager enabled the task to automatically manage the market balance of the miner's market actor (Default: false)
+	EnableMarketBalanceManager bool
```

Comment:
For separate PR, but we could create a separate balance manager, similar to the lotus-shed balancer command (https://github.com/filecoin-project/lotus/blob/master/cmd/lotus-shed/balancer.go): Could define groups with source and destination addresses, and have it automatically shuffle funds e.g. to commit/precommit control addresses